src/sync/sync_permission.cpp

```diff
@@ -261,6 +274,9 @@ SharedRealm Permissions::management_realm(std::shared_ptr<SyncUser> user, const
             Property::make("statusCode", PropertyType::Int, PrimaryKey::no, Indexed::no, Nullable::yes),
             Property::make("statusMessage", PropertyType::String, PrimaryKey::no, Indexed::no, Nullable::yes),
             Property::make("userId", PropertyType::String, PrimaryKey::no, Indexed::no, Nullable::no),
+            Property::make("metadataKey", PropertyType::String, PrimaryKey::no, Indexed::no, Nullable::yes),
+            Property::make("metadataValue", PropertyType::String, PrimaryKey::no, Indexed::no, Nullable::yes),
+            Property::make("metadataNamespace", PropertyType::String, PrimaryKey::no, Indexed::no, Nullable::yes),
```

Comment:
Is `metadataNamespace` or `metadataNameSpace` the correct spelling of this property name? realm/realm-object-server#1137 appears to use both, and I'm not clear which is relevant here.